zeppelin-web/app/scripts/app.js

```diff
@@ -100,9 +97,9 @@ angular
     'puElasticInput',
     'xeditable'
   ])
-  .filter('breakFilter', function() {
+  .filter('breakFilter', function () {
```

Comment:
Actually there is no space here.
Regarding the code style, we are using https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml,
or at least try to fix everything to match it (sometimes its hard to catch them all).